lib/pf/pfcmd/checkup.pm

```diff
@@ -1319,6 +1325,27 @@ sub pfdetect {
     }
 }
 
+=head2 validate_access_filters
+
+validate_access_filters
```

Comment:
something more descriptive would be nice

Reply:
how about 'Validate Access Filters'?